### PR TITLE
RetailCalendarWeek.weekOfQuarter, RetailCalendarWeek.quarterOfYear calculate correctly

### LINKED ISSUE #29: RetailCalendarWeek.weekOfQuarter and RetailCalendarWeek.quarterOfYear is not calculated correctly
With the merging of #22 , I introduced a bug in the calculation of the fields `weekOfQuarter` and `quarterOfYear` on `RetailCalendarWeek`

For the following setup:

```
    const calendar = new RetailCalendarFactory({
        weekCalculation: WeekCalculation.LastDayNearestEOM,
        weekGrouping: WeekGrouping.Group445,
        lastDayOfWeek: LastDayOfWeek.Friday,
        lastMonthOfYear: LastMonthOfYear.January,
        leapYearStrategy: LeapYearStrategy.AddToPenultimateMonth
      }, 2022)

    const week = calendar.weeks[39]
```

I *expect* `week.weekOfQuarter` to be `0`, however I *receive* `11`
I *expect* `week.quarterOfYear` to be `4` and I received something else

### PR DESCRIPTION
**Pull request by mikfreedman**
fixes #29

### CODE REVIEW

**Comment by mikfreedman** [comment]
@cihancimen can you please take a peek at this when you have a moment

File: src/retail_calendar.ts [hunk 1]
```diff
@@ -149,25 +149,29 @@ export const RetailCalendarFactory: RetailCalendarConstructor = class Calendar
     let weeksInQuarter = 0
```
Comment (cihancimen) [inline src/retail_calendar.ts:149]: This value is no longer read. Should be safe to remove.

**Review by cihancimen — COMMENTED** [review]
I am sorry for the late review.
There is a variable that is no longer used. Otherwise it looks good to me.

Thanks for the PR!

File: src/retail_calendar.ts (hunk 1 above)
Comment (mikfreedman) [reply src/retail_calendar.ts]: This is fixed!

**Comment by mikfreedman** [comment]
@cihancimen please review when you can

**Review by cihancimen — APPROVED** [review]

**Pull request merged**